src/storage_manifests.py

```diff
@@ -77,12 +76,7 @@ class UpdateSecrets(Patch):
 
     def __call__(self, obj):
         """Update the secret volume spec in daemonsets and deployments."""
-        if not any(
-            [
-                (obj.kind == "DaemonSet" and obj.metadata.name == "csi-cinder-nodeplugin"),
-                (obj.kind == "Deployment" and obj.metadata.name == "csi-cinder-controllerplugin"),
-            ]
-        ):
+        if not (obj.kind == "Deployment" and obj.metadata.name == "csi-cinder-controllerplugin"):
```

Comment:
Oops... i didn't do you well if i suggested this line change.  This shouldn't be changed.  It was totally incorrect before
```suggestion
        if not any(
            [
                (obj.kind == "DaemonSet" and obj.metadata.name == "csi-cinder-nodeplugin"),
                (obj.kind == "Deployment" and obj.metadata.name == "csi-cinder-controllerplugin"),
            ]
        ):
```

Reply:
gosh, my mistake